src/core/model/actions.ts

```diff
@@ -177,6 +177,6 @@ export const Actions = {
   addUnscopedElementToLayout,
   moveUiSchemaElement,
   removeUiSchemaElement,
-  setUiSchemaOptions,
+  updateUISchemaElement: setUiSchemaOptions,
```

Comment:
shouldn't we update the name of the `setUiSchemaOptions` method too?

Reply:
Fixed it, I trusted in the VS code rename action but it didn't do exactly what I assumed it would :)